website/pages/how-to-guides/_meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "*": {
+    "//": "Disable sidebar and search for these pages",
+    "theme": {
+      "sidebar": false,
+      "typesetting": "article"
+    },
+    "searchable": false
+  },
+  "cloud-asset-inventory-cloudquery-apache-superset": {
```

Comment:
What's the reason we need to explicitly set these?

Reply:
The reason is that I wanted the titles to be slightly different in the index than on the actual post. We have 5 posts that all start with `How to Build an Open Source Cloud Asset Inventory with` and the long titles made the index page look both repetitive and badly spaced. So in the index I called it `How Visualize CloudQuery data with...` to make it shorter but keep the original titles + url paths for SEO reasons.